playbook/app/pb_kits/playbook/pb_typeahead/typeahead.rb

```diff
@@ -40,6 +40,8 @@ class Typeahead < Playbook::KitBase
       prop :pill_color, type: Playbook::Props::Enum,
                         values: %w[primary neutral success warning error info data_1 data_2 data_3 data_4 data_5 data_6 data_7 data_8 windows siding roofing doors gutters solar insulation accessories],
                         default: "primary"
+      prop :required, type: Playbook::Props::Boolean,
```

Comment:
Does this allow both Boolean values and arbitrary strings? I think we would need to support both. True for when you want rails to use the attribute name / translations to generate a string, false when for whatever reason you do not want a label, and a string for when you have a specific value you would like to use for the label.

Reply:
I believe only the text input and text area kits (all the _field variations) support validation strings at this time. Going to add validation prop here and suggest adding that to other kits as well in follow up.